### PR TITLE
Integrate with HACS

### PR DESCRIPTION
**Pull request by vit-**
Make this repo installable via [HACS ](https://hacs.xyz/) by slightly changing the directories structure and adding a `hacs.json` file.

### CODE REVIEW

**Comment by thomasgermain** [comment]
Thanks for the good work !
Last time I check to integrate with HACS, I thought it was harder to do. Let me check the HACS doc

**Comment by vit-** [comment]
These are the minimal changes to make the component installable via hacs.

File: hacs.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "multimatic",
+  "domains": ["vrc", "vaillant", "multimatic"]
+}
```
Comment (thomasgermain) [inline hacs.json:4]: `zip_release` should be set to true, but then, I'm not sure on what to set in `filename` ? Zip in release are named: `vaillant-component-{release}`, so this is not static. Do you have any idea ?

`iot_class` should be `Cloud Polling`

File: hacs.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "multimatic",
+  "domains": ["vrc", "vaillant", "multimatic"]
```
Comment (thomasgermain) [inline hacs.json:3]: I think `domains` is the list of HA domains, so it should be: `["binary_sensor", "climate", "fan", "sensor", "water_heater"]`.

**Review by thomasgermain — CHANGES_REQUESTED** [review]
I also think the `manifest.json` should be adapted, see https://hacs.xyz/docs/publish/integration#manifestjson

I'm missing the `issue_tracker`, and I guess `documentation` should point to the readme

**Comment by thomasgermain** [comment]
Well yes, I remember now that I read the doc. There are 2 PRs to do (one to HACS and one to brands repository, for the logo). And there are lots of rules to fulfil in order to be accepted

File: hacs.json (hunk 1 above)
Comment (vit-) [reply hacs.json:4]: Other projects use `zip_release` when there's a built zip attached to the GitHub release, e. g. [alexa media player](https://github.com/custom-components/alexa_media_player/releases). In this repo, there's no such zip file attached. So, I don't think `zip_release` is applicable in this case.

Comment (thomasgermain) [reply hacs.json:5]: ha yeah, misunderstood the doc, sorry

**Comment by vit-** [comment]
> There are 2 PRs to do (one to HACS and one to brands repository, for the logo). And there are lots of rules to fulfil in order to be accepted

That is required if you want this component to be added to the official HACS integrations list. 
However, it is possible to install the component by adding a custom repository in HACS. `hacs.json` is the only requirement in this case.

**Comment by pepsonEL** [comment]
better will be official integration in HA.... no HACS....

**Comment by thomasgermain** [comment]
Just one last thing: I think `render_readme` should be set to true, otherwise there is no doc, see:
image
(this is basically saying there is nod doc available).

**Pull request merged**